.eslintrc.js

```diff
@@ -202,6 +204,12 @@ module.exports = {
     "semi-spacing": [2, {"before": false, "after": true}],
     "sort-imports": 2,
     "sort-vars": 2,
+    // Override this until eslint-plugin-mozilla gets updated.
+    "space-before-function-paren": ["error", {
+      "anonymous": "never",
+      "asyncArrow": "always",
```

Comment:
What will happen when `eslint-plugin-mozilla` gets updated? Looks like it will be "never" for everything, so maybe we should just match and autofix? https://searchfox.org/mozilla-central/source/tools/lint/eslint/eslint-plugin-mozilla/lib/configs/recommended.js#366-367

Reply:
ESLint 4 tweaked the async arrow option - it was ignored by default in v3. The v4 changes haven't landed in m-c yet, although I guess I could start landing parts of them soon.